x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/RestSqlQueryAction.java

```diff
@@ -58,15 +59,19 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
         }
 
         responseMediaType = sqlMediaTypeParser.getMediaType(request, sqlRequest);
-
+        if (responseMediaType == null) {
+            String msg = String.format(Locale.ROOT, "Invalid response content type. Accept=%s Content-Type=%s format=%s",
```

Comment:
```suggestion
            String msg = String.format(Locale.ROOT, "Invalid request content type: Accept=[%s], Content-Type=[%s], format=[%s]",
```

Generally QL code uses square brackets to enclose values; this makes white space easier to spot and formatting to read.
Also, I guess we want to point out that the _request_ is ill-formatted?

But otherwise LGTM, thanks.

Reply:
fixed.
We want to point out that the request is not known. Could be a valid format but unknown value.

Reply:
apologies, I misread your comment.
We want to point out that response content type - which his requested on request's Accept header or format parameter - is incorrect